src/fivetran-destination/src/destination/config.rs

```diff
@@ -146,6 +156,12 @@ pub async fn connect(
     let mut builder = SslConnector::builder(SslMethod::tls_client())?;
     builder.set_verify_cert_store(cert_store.build())?;
 
+    let options = if let Some(cluster) = cluster {
+        format!("--cluster={cluster}")
```

Comment:
I believe this needs to be escaped. From [libpq](https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNECT-OPTIONS):

> Spaces within this string are considered to separate command-line arguments, unless escaped with a backslash

Reply:
(I'm not 100% certain if tokio_postgres matches the libpq behavior here, so this might take a bit of investigation into what tokio_postgres does compared to libpq/psql.)

Reply:
I looked into it an `tokio_postgres` just writes the bytes from the `str` as a null-terminated c string, so no escaping! Added the proper escaping and tested manually to make sure it works